domain-specific-datasets/app/infer.py

```diff
@@ -0,0 +1,16 @@
+import os
```

Comment:
Things I mentioned earlier about inference and potentially difficult API/LLM usage etc.

A lot of community members don't seem to have money for LLM API access.